src/components/pages/employees/detail/General.tsx

```diff
@@ -177,11 +178,52 @@ export const General = (props: Props) => {
     mutate([GET_PATHS.getEmployees, data.username])
   }
 
+  const isKeepFwdEmailRef = useRef(false)
```

Comment:
Should use state instead of ref (`useState`)

Reply:
I was attempting to use the state hook, but it doesn’t work because of asynchronous data synchronization between the Modal and the main portal. Thanh suggested that I use the ref hook instead.